portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -61,23 +69,43 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     for (Entity entity : results.asIterable()) {
       if(i < commentNum || commentNum > commentList.size()) {
         String commentEntity = (String) entity.getProperty("comments");
-        commentList.add(commentEntity);
+        String fullcomment = nickname + ": " + commentEntity + " ";
```

Comment:
This is using the nickname of the current user, since nickname is set in line 51.

the user's nickname would need to be a property of the entity (or a user id that we can use to look it up, but, it's easier to just save it as a property)

Reply:
@SeanT18 I think this is the only bugfix of all the comments. The rest are style points.

I think this is going to be a non-trivial fix, so maybe just remove nickname from the display for now and proceed to week 5? I think that's higher-priority for now, and we can revisit this later.

Reply:
ok